src/asm3/wordprocessor.py

```diff
@@ -153,6 +153,9 @@ def additional_field_tags(dbo, fields, prefix = ""):
             val = af["ANIMALNAME"]
         if af["FIELDTYPE"] in human_flags:
             val = af["OWNERNAME"]
+            human = asm3.additional.get_human_record(dbo, int(af["VALUE"]))[0]
```

Comment:
Would prefer person_types to "human_flags" - they are additional field types, not flags!

As Saggi said, use asm3.person.get_person - no need to reinvent the wheel here and get_person resolves foreign keys for you